src/uta/models.py

```diff
@@ -225,7 +225,7 @@ class ExonAln(Base):
 class AssociatedAccessions(Base):
     __tablename__ = "associated_accessions"
     __table_args__ = (
-        sa.UniqueConstraint("origin", "tx_ac", "pro_ac", name="unique_pair_in_origin"),
+        sa.Index("unique_pair_in_origin", "origin", "tx_ac", "pro_ac", unique=True),
```

Comment:
This `Index` matches the definition of the existing database. And I think provides the same functionality.